applications/vanilla/models/class.commentmodel.php

```diff
@@ -465,7 +465,32 @@ public function lookup(array $where = [], $permissionFilter = true, $limit = nul
             return new Gdn_DataSet([]);
         }
 
-        $result = $this->getWhere($where, 'CommentID', $order, $limit, $offset);
+        // Avoid ambiguous fields.
+        $ambiguousFields = ['Body', 'DiscussionID', 'Format'];
```

Comment:
This has the potential to break if we do an update and will probably fail with `DateInserted` anyway.

I suggest that you pad all $where with 'c.' instead.